### DISCUSSION TITLE
No close Icon on p-message

### DISCUSSION THREAD

**Discussion by AlisiaRush** [primefaces/community#203] [PrimeNG] [answered]
I am trying to create a simple info box for my Angular app. I am usingAngular version 16. Whenever I try to use the <p-message> element, I do not see a closing icon. When I try to set closable to true, I get the following error.

Can't bind to 'closable' since it isn't a known property of 'p-message'.
 Here is an example of my code:
<p-message severity="info" text="{{ title }}" [closable]="true">

Here is an example of the results in the UI:
![image](https://github.com/primefaces/community/assets/6233240/1256ce99-bab5-4e77-bd7a-7483f1c504ef)

I have also tried the following because I need to create a title as well as a message.

<p-messages severity="info">
  <ng-template pTemplate>
    {{ title }}
    {{ message }}
  </ng-template>
</p-messages> 

This does not give me an info icon or a close icon. Please help

I have also imported these into my app.module.ts
import { MessageModule } from 'primeng/message';
import { MessagesModule } from 'primeng/messages';

**Comment by cetincakiroglu** [comment, marked as answer]
p-message doesn't have a closable property yet. It's a property of the p-messages component, you can use p-messages instead. For more info, please visit https://primeng.org/messages#closable

You can request this feature by opening a feature request issue on PrimeNG repo.

**Reply by AlisiaRush** [reply]
I have used the p-message as well. Here are the results:

![image](https://github.com/primefaces/community/assets/6233240/f444e5ce-feaf-4dc6-a4ac-481f29c8bdc4)
 
There seems to be a missing closeIcon here as well?

**Reply by AlisiaRush** [reply]
<p-messages [(value)]="title" [enableService]="false" [closable]="true"></p-messages>

**Reply by cetincakiroglu** [reply]
In the answer, it's suggested to use `p-messages` component instead of p-message. Both are different.

p-message is the inline message component and it doesn't have a closable property so you can't use it. Only `p-messages` and` toast `support closable property.

- Messages => wraps the p-message and inserts some extra visuals, for example close button and it has `closable` property

- p-message => inline message wrapped by messages component doesn't have `closable` property so you can't use it.
